web/src/lib/client/manager.test.js

```diff
@@ -60,3 +63,16 @@ describe("#startInstallation", () => {
     expect(managerProxy.Commit).toHaveBeenCalledWith();
   });
 });
+
+describe("#rebootSystem", () => {
+  beforeEach(() => {
+      cockpit.spawn = jest.fn().mockResolvedValue("");
+  });
+
+  it("returns whether the system reboot command was called or not", async () => {
+    const client = new ManagerClient(dbusClient);
+    const reboot = await client.rebootSystem();
+    expect(cockpit.spawn).toHaveBeenCalledWith(["/usr/sbin/shutdown", "-r", "now"])
+    expect(reboot).toEqual("");
```

Comment:
We might consider resolving to a boolean, so you know whether the command was successfully run.